src/config/handler.rs

```diff
@@ -1073,11 +1074,14 @@ impl ConfigProxyHandler {
             None => return Ok(()),
             Some(l) => l.to_ref(),
         };
+        let before = output
+            .find_workspace_insertion_point(&ws.name)
+            .map(|nr| nr.deref().clone());
         let config = WsMoveConfig {
             make_visible_always: false,
             make_visible_if_empty: true,
             source_is_destroyed: false,
-            before: None,
+            before,
         };
         move_ws_to_output(&link, &output, config);
```

Comment:
How about letting `move_ws_to_output` handle find the insertion point?

Reply:
Sure you want that changed? Couple of other things also depend on `move_ws_to_output` with `WsMoveConfig.before`.

Reply:
>Sure you want that changed?

Yes